src/codegen/index.ts

```diff
@@ -1,22 +1,23 @@
-import * as cg from "./tscodegen";
-import ApiGenerator from "./generate";
-import ts from "typescript";
-import SwaggerParser from "@apidevtools/swagger-parser";
-import converter from "swagger2openapi";
-import { OpenAPIV3 } from "openapi-types";
+import * as cg from './tscodegen';
```

Comment:
None of the changes in this file are functional but purely stylistic, can you remove them?

Reply:
Done.